blindsign/blindrsa/internal/common/rsa.go

```diff
@@ -107,7 +106,7 @@ func decrypt(random io.Reader, priv *keys.BigPrivateKey, c *big.Int) (m *big.Int
 		for {
 			r, err = rand.Int(random, priv.Pk.N)
 			if err != nil {
-				return
+				return nil, err
```

Comment:
The above two changes address:
```
::error file=blindsign/blindrsa/internal/common/rsa.go,line=92,col=3::naked return in func `decrypt` with 48 lines of code (nakedret)
::error file=blindsign/blindrsa/internal/common/rsa.go,line=110,col=5::naked return in func `decrypt` with 48 lines of code (nakedret)
```

`m` has not been initialized up to this point, and `err` has been initialized. There are no writes to `m` in this loop, so returning `nil` is alright.